### PR TITLE
Added info on ntlm and credssp, updated configure script for credssp

### PR DESCRIPTION
**Pull request by jborean93**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
intro_windows.rst

##### ANSIBLE VERSION
```
ansible 2.2.1.0
  config file = /cygdrive/d/dev/mr-ansible/ansible.cfg
  configured module search path = Default w/o overrides
```

##### SUMMARY
This PR adds in more info regarding authentication options for Windows that were missing before such as NTLM and CredSSP. It has also changed the ConfigureRemotingForAnsible.ps1 to setup CredSSP on the host.

### CODE REVIEW

**Comment by jborean93** [comment]
Hey @dagwieers I saw your post on https://github.com/ansible/ansible/pull/17851 and thought it might be best to at least add in some info regarding NTLM and other auth methods. Hopefully this is alright to merge in and have enough info until @nitzmahone overhauls it.

**Comment by nitzmahone** [comment]
Hey @jborean93, are you on IRC? If so, can you PM me (`@nitzmahone` on freenode)?

**Comment by jborean93** [comment]
@nitzmahone sorry I can't use IRC where I am at right now.

**Comment by dagwieers** [comment]
@jborean93 Looking good, but I am not an expert.
The overview of authentication methods and what they allow is very interesting.
But I will let @nitzmahone and the other windows devs decide :-)

**Comment by nitzmahone** [comment]
@jborean93 No worries- you've got mail. ;)

File: examples/scripts/ConfigureRemotingForAnsible.ps1 [hunk 1]
```diff
@@ -258,6 +259,15 @@ Else
     Write-Verbose "Basic auth is already enabled."
 }
 
+# Check for CredSSP authentication
```
Comment (nitzmahone) [inline examples/scripts/ConfigureRemotingForAnsible.ps1]: This is great, but would prefer if it were a switch- some places are actively against CredSSP, so we probably don't want to enable by default (though one could make the same argument about Basic I suppose).

**Comment by nitzmahone** [comment]
... and yeah, this is a much lighter-weight version of what I've had back-burnered for awhile now- I'm happy to merge since you went to the trouble of writing it within the mess that is the current Windows docs, as it's *far* better than the vacuum that's there now about the newish auth stuff. If you want to make the change for the switch and/or split that script stuff to another PR, I'm good to merge.

File: examples/scripts/ConfigureRemotingForAnsible.ps1 (hunk 1 above)
Comment (jborean93) [reply examples/scripts/ConfigureRemotingForAnsible.ps1]: Have added a switch for CredSSP and added the info to the docs

**Review by nitzmahone — APPROVED** [review]

**Comment by nitzmahone** [comment]
Thanks!

**Pull request merged**